script/utils/api.js

```diff
@@ -0,0 +1,54 @@
+const CODEIT_URL = `https://bootcamp-api.codeit.kr/api`;
+
+export const signInAccess = async (email, password) => {
+  const response = await fetch(`${CODEIT_URL}/sign-in`, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify({
+      email: email,
+      password: password,
+    }),
+  });
+  if (!response.ok) {
+    throw new Error('로그인에 실패했습니다.');
+  }
+  const body = await response.json();
+  return body;
```

Comment:
현재 코드도 잘못된 부분은 없으나 간결성을 위해 요렇게 줄여서 작성해볼 수도 있겠네요 :) 
(async 함수 내 return 문에는 await 없이 작성 가능합니다) 

```suggestion
  return response.json();
```

Reply:
await 구문이 없어도 된다니 신기하네요! 참고해보겠습니다!